src/utils/SVGIcons.ts

```diff
@@ -10,6 +10,8 @@ export class SVGIcons {
     arrowDown: require(`svg/arrow-down`),
 
     mainClear: require(`svg/main-clear`),
+    clearSmall: require(`svg/clear-small`),
+    checkYes: require(`svg/check-yes`),
```

Comment:
This icon was provided by @acverrette and would need to be added to Vapor if we use it.

Reply:
Why do we need to add it to react vapor ? If they are not using in in their project ?

I don't think that's really necessary.

Reply:
It's only because it would be useful to have it there since I can see possible uses of this icon in the admin. But indeed, that should not be the case for all JSUI icons. It's not mandatory to add it in vapor :).